lightning/src/chain/package.rs

```diff
@@ -686,14 +717,42 @@ impl PackageTemplate {
 		}
 		None
 	}
+
+	/// Computes a feerate based on the given confirmation target. If a previous feerate was used,
+	/// and the new feerate is below it, we'll use a 25% increase of the previous feerate instead of
+	/// the new one.
+	pub(crate) fn compute_package_feerate<F: Deref>(
+		&self, fee_estimator: &LowerBoundedFeeEstimator<F>, conf_target: ConfirmationTarget,
+	) -> u32 where F::Target: FeeEstimator {
+		let feerate_estimate = fee_estimator.bounded_sat_per_1000_weight(conf_target);
+		if self.feerate_previous != 0 {
+			// If old feerate inferior to actual one given back by Fee Estimator, use it to compute new fee...
+			if feerate_estimate as u64 > self.feerate_previous {
+				feerate_estimate
+			} else {
+				// ...else just increase the previous feerate by 25% (because that's a nice number)
```

Comment:
> (because that's a nice number)

Just something that works okay or does another implementation do this? :sweat_smile:

Reply:
There is no BOLT recommendation on a compelling fee-bumping strategy for time-sensitive closes. Ideally, we would rely on a smart fee-estimator telling us the needed feerate increase to stay in top block of network mempools, though that type of fee-estimation doesn't exist yet.